backend/load_test/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM debian:buster
+
+RUN apt update
+RUN echo yes | apt upgrade
+
+RUN echo yes | apt install nodejs npm curl git
+
+RUN npm install -g n 
+RUN n latest
+
+RUN npm install -g artillery
+
+CMD ["/bin/bash"]
+# docker run -ti -v $(pwd)/script/:/srv/script load_artillery
```

Comment:
What is this comment ? What purpose does it serve?

Reply:
was test purpose on creation